.pip_install_for_tests.sh

```diff
@@ -18,3 +18,16 @@ do
         pip3 install --user $package
     fi
 done
+
+# Install xBOUT to make sure we get master version
+git clone https://github.com/boutproject/xBOUT.git
+cd xBOUT
+pip3 install --user .
+cd ..
+
+# Install Hermes-3
+git clone https://github.com/boutproject/xhermes.git
+cd xhermes
+pip3 install --user .
+cd ..
```

Comment:
If you want to do this, I would suggest:
```suggestion
pip install  --user git+https://github.com/boutproject/xBOUT.git
pip install  --user git+https://github.com/boutproject/xhermes.git
```

But I think, if there are changes that are wanted / needed, there should be a release. Otherwise everybody will always start installing from git, which is imho not as nice, as using the released and version package.

Releases are cheap enough, so lets use them!

Reply:
At the moment I need the latest master of XBOUT so that the grid loading could function properly in the test. This was my workaround, since I didn't know how to do that through pip. Happy to refactor as suggested if @bendudson @mikekryjak agree on the style change -- I didn't invent the xhermes line.

Reply:
The style change seems equivalent - if that is the case then sure, why not.
@dschwoerer I agree with you with regards to releases. I've had practically zero time to contribute to xHermes for a while. We may be getting some RSE support that could help with this in the next two months.